gdcdictionary/schemas/exposure.yaml

```diff
@@ -38,15 +38,31 @@ uniqueKeys:
   - [project_id, submitter_id]
 
 deprecated:
-  - bmi
-  - height
-  - weight
+  - smokeless_tobacco_quit_age
+  - asbestos_exposure
+  - radon_exposure
+  - respirable_crystalline_silica_exposure
+  - years_smoked
```

Comment:
are we supposed to be removing years_smoked or deprecating it?

Reply:
@wwysoc2 said that the migration for this property wasn't done, so instead of removing it we need to add it to the deprecated list.